### PR TITLE
UI rework + post release bugfixes

### PR DESCRIPTION
**Pull request by joelAVS**
The entire green ui has been redone. Plus a pletora of bug fixes and balance changes

### CODE REVIEW

File: TGC/common/buildings.txt [hunk 1]
```diff
@@ -732,15 +732,15 @@
 		# Audax Validator "." Ignore_NEXT
 		type = university
 		# Audax Validator "." Ignore_NEXT
-		cost = 30000
+		cost = 15000
```
Comment (ColdSlav) [inline TGC/common/buildings.txt:735]: What is the purpose of this change? I require clarification

File: TGC/common/buildings.txt [hunk 2]
```diff
@@ -732,15 +732,15 @@
...
 		goods_cost =
 		{
-			steel = 20
+			steel = 30
```
Comment (ColdSlav) [inline TGC/common/buildings.txt:738]: Same for this one

File: TGC/common/buildings.txt [hunk 3]
```diff
@@ -732,15 +732,15 @@
...
 			lumber = 90
 			cement = 90
-			paper = 10
-			dye = 10
-			glass = 10
+			paper = 20
+			#dye = 10
```
Comment (ColdSlav) [inline TGC/common/buildings.txt:742]: These ones too

File: TGC/common/buildings.txt [hunk 4]
```diff
@@ -750,9 +750,10 @@
 		province = yes
 		# Audax Validator "." Ignore_NEXT
 		one_per_state = yes
+		pop_build_factory = yes
```
Comment (ColdSlav) [inline TGC/common/buildings.txt:753]: Missing line perhaps?

File: TGC/common/buildings.txt [hunk 5]
```diff
@@ -750,9 +750,10 @@
...
 
-		research_points = 0.2
-
+		research_points_modifier = 0.025
```
Comment (ColdSlav) [inline TGC/common/buildings.txt:755]: Clarify

File: TGC/history/provinces/low countries/377 - Utrecht.txt [hunk 6]
```diff
@@ -4,5 +4,3 @@ add_core = NET
 trade_goods = grain
 life_rating = 35
 #terrain = 
-
-university = 1
```
Comment (ColdSlav) [inline TGC/history/provinces/low countries/377 - Utrecht.txt:8]: Is there a reason this was removed?

**Review by ColdSlav — COMMENTED** [review]
Good job so far, though there are some changes I need you to clarify.

File: TGC/common/buildings.txt (hunk 5 above)
Comment (joelAVS) [reply TGC/common/buildings.txt:755]: change log

File: TGC/common/buildings.txt (hunk 1 above)
Comment (joelAVS) [reply TGC/common/buildings.txt:735]: change log

File: TGC/common/event_modifiers.txt [hunk 7]
```diff
@@ -5315,4 +5315,4 @@ in_disrepair = {
 
 call_ally_cooldown = {
 	icon = 1
-}
+}
```
Comment (GeneralVonGelre) [inline TGC/common/event_modifiers.txt:5318]: Is this the file encoding getting messed up?

Might be so make sure to check

File: TGC/decisions/NET.txt [hunk 8]
```diff
@@ -355,6 +355,9 @@ political_decisions = {
 			badboy = 5
 			set_country_flag = artois_claimed
 			FRA_404 = { add_core = NET }
+			405 = { add_core = NET }
+			400 = { add_core = NET }
+			399 = { add_core = NET }
```
Comment (GeneralVonGelre) [inline TGC/decisions/NET.txt:360]: might be cleaner to do this by scoping the respective states of these provinces and adding a Dutch core to all the French cores in those regions.

File: TGC/common/buildings.txt (hunk 3 above)
Comment (joelAVS) [reply TGC/common/buildings.txt:742]: Removed due UI ugliness, it cant fit more than 4 resources properly

File: TGC/common/buildings.txt (hunk 4 above)
Comment (joelAVS) [reply TGC/common/buildings.txt:753]: No, intentional. I've balanced around that

File: TGC/history/provinces/low countries/377 - Utrecht.txt (hunk 6 above)
Comment (joelAVS) [reply TGC/history/provinces/low countries/377 - Utrecht.txt:8]: Double in the same state by accident.
